db/migrate/20230712140250_add_fields_to_comments.rb

```diff
@@ -0,0 +1,7 @@
+class AddFieldsToComments < ActiveRecord::Migration[7.0]
+  def change
+    # add_column :comment, :author_id, :integer
+    # add_column :comment, :post_id, :integer
+    # add_column :comment, :text, :text
+  end
```

Comment:
- [ ] I observed you newly added commented-out codes after the last review, kindly ensure that the migration file is not empty for smooth running of the database. 👍